lib/experiments/index.js

```diff
@@ -6,6 +6,7 @@ const symbol = require('../utilities/symbol');
 let experiments = {
   MODULE_UNIFICATION: symbol('module-unification'),
   DELAYED_TRANSPILATION: symbol('delayed-transpilation'),
+  BROCCOLI_MASTER: process.env.BROCCOLI_MASTER === '1', // Note: EXPERIMENTAL, DO NOT USE!
```

Comment:
Hehe, these are all experimental 😉

Reply:
Lets make the flag: `process.env.EMBER_CLI_BROCCOLI_1` (I don't like using `_MASTER` conceptually here)...